_includes/configuration/advtable_value_series.md

```diff
@@ -0,0 +1,89 @@
+## `advtable_value_series`
+{{ site.requires_5_9v }}
+
+{% if pluginname != "Advanced Tables" %}
```

Comment:
Why do we have this? I can't see this being included anywhere else in this PR.

Reply:
I just kept it from the template file. I'll remove it since this wouldn't be used outside of the advtable page. Thanks.